File: WinForms/VB/GridMainDemo/ModulesLayoutView/BaseLayoutModule.vb
Imports System.Data

Namespace DevExpress.XtraGrid.Demos

    Public Class BaseLayoutModule
        Inherits TutorialControl

        Protected Overrides Sub InitXMLData(ByVal dataFileName As String, ByVal ds As DataSet)
            ds.ReadXml(dataFileName)
        End Sub

        Protected Overrides ReadOnly Property ModulesFolder As String
            Get
                Return "GridMainDemo\ModulesLayoutView\"
            End Get
        End Property
    End Class
End Namespace

File: WinForms/CS/GridMainDemo/ModulesLayoutView/BaseLayoutModule.cs
using System.Data;

namespace DevExpress.XtraGrid.Demos {
    public class BaseLayoutModule : TutorialControl {
        protected override void InitXMLData(string dataFileName, DataSet ds) {
            ds.ReadXml(dataFileName);
        }
        protected override string ModulesFolder {
            get { return "GridMainDemo\\ModulesLayoutView\\"; }
        }
    }
}
